src/util.js

```diff
@@ -0,0 +1,7 @@
+export const errorString = (error) => `[ERROR] ${error}`;
+
+export const isNumber = (num) => !Number.isNaN(num);
+
+export const isPositiveNumber = (num) => num >= 0;
```

Comment:
```suggestion
export const isPosisiveNumber = (num) => num > 0;
```
과제의 요구사항에서 양수의 총합을 계산하라 하였기 때문에 0은 제외해주셔야 할 것 같습니다.

Reply:
앗 실수했네요 감사합니다 ㅎㅎ